jdaviz/app.vue

```diff
@@ -38,7 +38,8 @@
       <v-toolbar-items v-for="item in state.tool_items">
         <v-divider v-if="['g-data-tools', 'g-subset-tools'].indexOf(item.name) === -1" vertical style="margin: 0px 10px"></v-divider>
         <v-divider v-else-if="item.name === 'g-subset-tools'" vertical style="margin: 0px 10px; border-width: 0"></v-divider>
-        <j-tooltip :tipid="item.name">
+        <j-tooltip v-if="config == 'cubeviz' && item.name == 'g-data-tools' && state.data_items.length !== 0"></j-tooltip>
+        <j-tooltip v-else :tipid="item.name">
```

Comment:
could probably either make the first (empty) `v-if` entry a div instead of a j-tooltip and/or invert the v-if and not render any entry in the case of hiding the import button (but that might be harder logic to parse).

Reply:
Giving both the same element seemed more intuitive to me. I did want to put the conditional for hiding in the `v-else` statement since it's rarer, but the logic for the `v-if` seems like it would need to be long – I get something like `v-if="(config !== 'cubeviz') || ((config == 'cubeviz') && (item.name !== 'g-data-tools' || (item.name == 'g-data-tools' && state.data_items.length == 0)))"`.